.gitignore

```diff
@@ -30,3 +30,4 @@ yarn-debug.log*
 yarn-error.log*
 
 storybook-static
+.vscode/settings.json
```

Comment:
@tinkertrain Whoa! How come? This was intentional :D. We want shared project settings